js/api/Surveys.js

```diff
@@ -1,88 +1,92 @@
+
+import { loadForms, loadParseFormDataBySurveyId } from './Forms';
+import { checkSurveyTimeTriggers, removeTriggers } from './Triggers';
+import { loadInvitations, loadAcceptedInvitations } from '../api/Invitations';
 
 // Attempts to find a survey with a specified id cached in the Store
 export function loadCachedSurvey(id) {
-  return realm.objects('Survey').filtered(`id = "${id}"`)[0]
+  return realm.objects('Survey').filtered(`id = "${id}"`)[0];
 }
 
 // Finds and returns the list of all surveys cached in Realm.io
 export function loadCachedSurveyList() {
-  return realm.objects('Survey')
+  return realm.objects('Survey');
 }
 
 // Finds and returns the list of all surveys cached in Realm.io
 export function loadAllAcceptedSurveys(callback) {
   loadAcceptedInvitations((err, invitations) => {
     if (err) {
-      callback(err, [])
-      return
+      callback(err, []);
+      return;
     }
 
-    const surveys = []
+    const surveys = [];
     const invitationsLength = invitations.length;
-    for (var i = 0; i < invitationsLength; i++) {
-      let acceptedSurvey = realm.objects('Survey').filtered(`id = "${invitations[i].surveyId}"`)[0];
+    for (let i = 0; i < invitationsLength; i++) {
+      const acceptedSurvey = realm.objects('Survey').filtered(`id = "${invitations[i].surveyId}"`)[0];
       if (acceptedSurvey) {
-        surveys.push(acceptedSurvey)
+        surveys.push(acceptedSurvey);
       }
     }
-    callback(null, surveys)
-  })
+    callback(null, surveys);
+  });
 }
 
 /**
  * Gets the Forms from a specified Survey from Parse
  * @param  {string}   surveyId ID of the target survey to fetch from
  * @param  {Function} callback Callback function which returns an array of Parse 'Form' objects
  */
-export function getSurveyForms(surveyId, callback){
-  const Survey = Parse.Object.extend("Survey")
-  const query = new Parse.Query(Survey)
-  query.get(surveyId, {
-    success: function(survey) {
-      loadForms(survey, function(err, forms){
-        if (callback) callback(null, forms)
-      })
+export function getSurveyForms(surveyId, callback) {
+  const Survey = Parse.Object.extend('Survey');
+  const query = new Parse.Query(Survey);
+  query.get(surveyId,
+    (survey) => {
+      loadForms(survey, (err, forms) => {
+        if (callback) {
+          callback(null, forms);
+        }
+      });
     }
-  })
+  );
 }
 
 // Queries the connected Parse server for a list of Surveys.
 export function loadSurveys(callback) {
-  const Survey = Parse.Object.extend("Survey")
-  const query = new Parse.Query(Survey)
-  query.equalTo("active", true)
+  const Survey = Parse.Object.extend('Survey');
+  const query = new Parse.Query(Survey);
+  query.equalTo('active', true);
   query.find({
-    success: function(results) {
-      clearSurveyCache(results)
-      let cachedSurveys = realm.objects('Survey')
+    success: (results) => {
+      clearSurveyCache(results); // eslint-disable-line no-use-before-define
```

Comment:
This is the linter rule forbidding hoisting, and was failing on my local `npm test`
This doesn't seem to be the case on circleci.
Are the eslint rules unsynced for this project?